polaris-react/src/components/KeyboardKey/KeyboardKey.tsx

```diff
@@ -1,15 +1,20 @@
 import React from 'react';
 
+import {classNames} from '../../utilities/css';
+
 import styles from './KeyboardKey.scss';
 
+type Variant = 'small';
+
 export interface KeyboardKeyProps {
-  /** The content to display inside the key */
   children?: string;
+  variant?: Variant;
```

Comment:
Looking across Polaris, seems like `size` would be a more appropriate prop name?